BST.rb

```diff
@@ -0,0 +1,320 @@
+load "node.rb"
+
+class Bst
+
+	def initialize()
```

Comment:
no need for () if you are not passing params

Reply:
Done